es/utils/aepp-wallet-communication/connection/browser-runtime.js

```diff
@@ -55,10 +55,10 @@ function disconnect () {
  */
 function connect (onMessage, onDisconnect) {
   if (this.isConnected()) throw new Error('You already connected')
-  this.handler = ((msg, source) => {
+  this.handler = (msg, source) => {
     if (this.debug) console.log('Receive message: ', msg)
     onMessage(msg, source)
-  }).bind(this)
+  }
```

Comment:
sure this is not breaking?

Reply:
Yes, https://stackoverflow.com/a/33308151/6176994
Also, I have checked it in the browser:
```js
> (() => this).bind({ test: true })()
Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}
> (function() { return this }).bind({ test: true })()
{ test: true }
```
It looks like a tricky question at the interview 😅